docs/user-guide/migrate-to-greptimedb/migrate-from-clickhouse.md

```diff
@@ -0,0 +1,237 @@
+---
+
+# Migrate from ClickHouse
+
+This guide provides a detailed explanation on how to smoothly migrate your business from ClickHouse to GreptimeDB. It covers pre-migration preparation, data model adjustments, table structure reconstruction, dual-write assurance, and specific methods for data export and import, aiming to achieve a seamless system transition.
+
+
+## Pre-Migration Notes
+
+- **Compatibility**
+  Although GreptimeDB is SQL protocol compatible, the two databases have fundamental differences in data modeling, index design, and compression mechanisms. Be sure to refer to the [SQL compatibility](/reference/sql/compatibility.md) documentation and official [modeling guidelines](/user-guide/administration/design-table.md) to refactor table structures and data flows during migration.
+
+- **Data Model Differences**
+  ClickHouse is a general-purpose big data analytics engine, while GreptimeDB is optimized for time-series, metrics, and log observability scenarios. There are differences in their data models, index systems, and compression algorithms, so it’s important to take these differences and the actual business scenario into account during model design and compatibility considerations.
+
+---
+
+## Refactoring Data Model and Table Structure
+
+**1. Time Index**
+- ClickHouse tables do not always have a `time index` field. During migration, you need to clearly select the primary time field for your business to serve as the time index and specify it when creating the table in GreptimeDB. For example, typical log or trace print times.
+- The time precision (such as second, millisecond, microsecond, etc.) should be assessed based on real-world requirements and cannot be changed once set.
+
+**2. Choose Between Tag and Field Columns**
+- Selecting tag columns (TAG): It is recommended to use highly reused and frequently filtered fields—such as host name, service name, region, or instance ID—as tag columns.
+  - Tags are suitable for low-cardinality and well-categorized fields, and can significantly accelerate filtering.
+- Field (data column): Actual observations, metrics, log message fields, etc. are suitable as fields; high-cardinality fields should be avoided for use as primary keys or tags.
+
+**3. Primary Key and Wide Table Recommendations**
+- Primary Key: Combine tag columns. It’s not recommended to include high-cardinality fields such as log IDs, user IDs or UUIDs to avoid primary key bloat, excessive write amplification, and inefficient queries.
+- Wide Table vs. Multiple Tables: For multiple metrics collected at the same observation point (such as on the same host), it’s better to use a wide table, which improves batch write efficiency and compression ratio.
+
+**4. Index Planning**
+- Inverted Index: Build indexes for low-cardinality tag columns to improve filter efficiency.
+- Skipping/Fulltext Index: Use as needed; avoid building unnecessary indexes on high-cardinality or highly variable fields.
```

Comment:
Describe skipping index using clickhouse equivalent concepts.

Reply:
We should also use seperated bullet points for skipping and fulltext index: describe what it is, not how to use it here.

Reply:
I don't think so ,the user must read our data index documents at first. Don't repeat the content here again.

Reply:
Addressed